bundles/org.openhab.binding.ekey/src/main/java/org/openhab/binding/ekey/internal/api/EkeyUdpPacketReceiver.java

```diff
@@ -48,10 +48,10 @@ public class EkeyUdpPacketReceiver {
 
     private boolean connected = false;
 
-    public EkeyUdpPacketReceiver(final String ipAddress, final int port, final String readerThreadName) {
+    public EkeyUdpPacketReceiver(final String ipAddress, final int port, final String threadName) {
```

Comment:
I would suggest to align with member variable name and fully revert. I hope you don't mind that I just commit this tiny change, since shortening the name in itself doesn't add any value, and the code will simply as a result be left unchanged:
```suggestion
    public EkeyUdpPacketReceiver(final String ipAddress, final int port, final String readerThreadName) {
```